lib/ControlServer.cpp

```
@@ -385,18 +381,32 @@ ControlServer::~ControlServer()
 
 void ControlServer::start()
 {
-    // listen to any interface using ipv4 or ipv6
-    auto const address = net::ip::make_address("::");
+    try {
```

Shouldn't we propagate the exception if one occurs?